act-framework.bs

```diff
@@ -86,8 +86,34 @@ ACT Output Data {#output}
 ACT Data Format {#output-format}
 --------------------------------
 
-Editor note: This section describes the required properties of the data output by a rule. Certain parts must be standardized to enable aggregation of results produced by different accessibility test tools. Additionally, standardizing parts of the output format is required for validating the implementation (see below). Data in the output format has to be accessible.
+Rules are tested in two steps. Firstly, the selector works out what elements / tags or other 'components' to test, these are known as the *Test Target*. Following this, each test target will be taken through the test procedure of the rule. This will give a *Test Result* for each *Test Target*.
 
+In addition to Test Target and Test Result, contextual information MUST be provided to indicate what web page, file or other test subject the rule was applied to, as well as an identifier of the rule itself. Running a rule will result in at least the following four pieces of information:
+
+- Test Criterion: The rule (or other test) that was used to generate the result
+- Subject: The web page or other component that was tested
+- Target: An element / component contained within the subject, that was selected for testing
+- Result: The finding as determined by the test, either: Pass, Fail or Unknown
+
```

Comment:
how is the Target identified? there can be several ways: DOM node, unique CSS selector, XPath, etc. Does it need to be specified here?